resources/public/api/conf/1.0/testdata/fund-release.md

```diff
@@ -0,0 +1,446 @@
+<table>
+    <col width="20%">
+    <col width="40%">
+    <col width="40%">
+    <thead>
+        <tr>
+            <th>Scenario</th>
+            <th>Request Payload</th>
+            <th>Response</th>
+        </tr>
+        <tr>
+              <td>
+                  <p>Fund release created</p>
+                  <p class="code--block">
+                  <strong>lisaManagerReferenceNumber:</strong><br>
+                  <a href="https://test-developer.service.hmrc.gov.uk/api-documentation/docs/api/service/lisa-api/1.0#testing-the-api">Use your test user profile</a><br>
+                   <br>
+                        <strong>accountId:</strong><br>1234567890
+                  </p>
+               </td>
+         <td>
+             <pre class="code--block">
+ {
+   "eventDate": "2017-05-10",
+   "withdrawalAmount": 4000.00,
+   "conveyancerReference": "CR12345-6789",
+   "propertyDetails": {
+          "nameOrNumber": "1",
+          "postalCode": "AA11 1AA",
+     }
+ }               
+             </pre>
+          </td>
+          <td>
+              <p>HTTP status: <code class="code--slim">201 (Created)</code></p>
+                <pre class="code--block">
+ {
+   "status": 201,
+   "success": true,
+   "data": {
+          "message": "Fund release created",
+          "fundReleaseId": "3456789000"
+    }
+ }               
+                        </pre>         
+                    </td>
+                </tr>
+                <tr>
+                     <td>
+                         <p>Fund release superseded</p>
+                         <p class="code--block">
+                            <strong>lisaManagerReferenceNumber:</strong><br>
+                            <a href="https://test-developer.service.hmrc.gov.uk/api-documentation/docs/api/service/lisa-api/1.0#testing-the-api">Use your test user profile</a><br>
+                               <br>
+                                   <strong>accountId:</strong><br>1234567890
+                           </p>
+                       </td>
+                       <td>
+        <pre class="code--block">
+ {
+   "eventDate": "2017-05-05",
+   "withdrawalAmount": 4000.00,
```

Comment:
Sorry should have noticed this earlier, but in this example the superseded details (withdrawalAmount, eventDate) are the same as the original. Can they be something different to show that they can be changed?